api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -1024,6 +1034,51 @@ public void testCloneWorkspaceDifferentOwner() throws Exception {
     assertThat(workspace2.getUserRoles().get(0).getEmail()).isEqualTo(cloner.getEmail());
   }
 
+  @Test
+  public void testCloneWorkspaceIncludeUserRoles() throws Exception {
+    User cloner = createUser("cloner@gmail.com");
+    createUser("reader@gmail.com");
```

Comment:
Can we abstract these strings to be reusable variables rather than hardcoding the string multiple times.

Reply:
Done